benchmarking_sbi/requirements.txt

```diff
@@ -1,7 +1,7 @@
 ansible
 boto
 hiplot
-hydra-core==1.0.3
+hydra-core>=1.0.3
```

Comment:
Should we make it `>=1.1.1` to be on the safe side? See https://github.com/facebookresearch/hydra/releases. We could also consider pinning Hydra and plugins to avoid incompatibility issues like this one in the future..

Reply:
for the time being yes. On the long run, this won't save us any headaches.

Reply:
Done [`3304992` (#3)](https://github.com/sbi-benchmark/results/pull/3/commits/33049922d584ee900f2a7bc7437067caff9ee555)